examples/asa/scripts/transfer/gold-delegated-lsig.debug.js

```diff
@@ -0,0 +1,53 @@
+/**
```

Comment:
could you please add a run command for this in example workflow ?

Reply:
ok